pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -697,6 +783,16 @@ func (k *Kubernetes) Transform(komposeObject kobject.KomposeObject, opt kobject.
 	// this will hold all the converted data
 	var allobjects []runtime.Object
 
+	if komposeObject.Secrets != nil {
+		secrets, err := k.CreateSecrets(komposeObject)
+		if err != nil {
+			return nil, errors.Wrapf(err, "create secrets error")
```

Comment:
Maybe add something different such as "Unable to create secrets" or something more descriptive? Not too sure what type of error message to add.